### PR TITLE
[feat] Enable listening on v6

### PR DESCRIPTION
**Pull request by chhetripradeep**
This allows enabling/disabling v4 and v6.

### CODE REVIEW

**Comment by hagen1778** [comment]
Hi @chhetripradeep! Thanks for PR! Why would someone need to disable listening on ipv4?

**Comment by chhetripradeep** [comment]
Hello @hagen1778 
Thank you for looking at it. I can think of scenarios where people only enable IPv6 throughout their internal network. If you think that i should remove the flag `enableTCP4` i will update it.

**Comment by hagen1778** [comment]
I'd vote for removing `enableTCP4` flag. This should simplify the PR significantly. Thank you!

File: go.mod [hunk 1]
```diff
@@ -5,6 +5,8 @@ go 1.13
 require (
 	github.com/DataDog/zstd v1.4.4
 	github.com/frankban/quicktest v1.7.2 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```
Comment (hagen1778) [inline go.mod]: btw, looks like this shouldn't be related to the change

**Comment by gontarzpawel** [comment]
Hi @chhetripradeep,
I've just merged changes allowing configurable IPv6 support #133 . Therefore, I'll close this PR

**Pull request closed without merging**